tests/unit/s2n_ssl_prf_test.c

```diff
@@ -22,8 +22,9 @@
 #include "testlib/s2n_testlib.h"
 #include "tls/s2n_prf.h"
 
-/*
```

Comment:
Any idea why this diff looks weird?

Reply:
I have my suspicions. There was double empty lines between the includes and the comment. Now it's just a single empty line, so the diff threw a fit. It's slightly more obvious in split diff mode.